app/src/main/java/com/example/golocal/fragments/ProfileFragment.java

```
@@ -103,6 +108,14 @@ public void onClick(View v) {
                     @Override
                     public void onClick(View v) {
                         user.put(KEY_BIO, etBio.getText().toString());
+                        user.saveInBackground(new SaveCallback() {
+                            @Override
+                            public void done(ParseException e) {
+                                if (e != null) {
+                                    Log.e(TAG, "error updating bio", e);
+                                }
```

we should tell the user that saving their bio failed and they should try again instead of just silently failing. Also, since it failed to save, we shouldn't update the UI with new text - it gives user the false impression that updating their bio succeeded